docs/src/main/sphinx/release/release-386.md

```diff
@@ -0,0 +1,61 @@
+# Release 386 (15 June 2022)
+
+## General
+
+* Improve fault-tolerant execution performance by updating default configuration
+  values. ({issue}`12646`)
+* Improve exchange spooling file enumeration speed. ({issue}`12615`)
+* Remove support for Phoenix 4. ({issue}`12772`)
+
+## JDBC driver
+
+* Add configuration property `assumeLiteralUnderscoreInMetadataCallsForNonConformingClients`
```

Comment:
hahah .. thats a terrible choice of a property name

Reply:
also .. do we need to make this more user facing .. or is this purely a developer facing property .. and do we have docs for this?

Reply:
It does have documentation, yeah. They had to make the table wider to fit the property in, but they did it! 😆